scripts/client.js

```diff
@@ -201,7 +201,20 @@ const processFile = async (asset) => {
 	if (ONLY_CACHE_JS && !asset.endsWith(".js")) return [];
 
 	const url = `${BASE_URL}/assets/${asset}`;
-	const res = await fetch(url, { agent });
+	let res,
+		tries = 5;
+	do {
```

Comment:
Doesn't this raise the min nodejs ver even higher?

Reply:
ah no, node-fetch is still used. to be clear this PR does not actually change this script, theyve just included my commit in their pr